Source/objects.cpp

```diff
@@ -987,7 +987,7 @@ void DeleteObject(int oi, int i)
 
 void AddChest(int i, int t)
 {
-	if (GenerateRnd(2) == 0)
+	if (flipCoin())

```

Comment:
I'm guessing this wasn't deliberate as it is the only one you didn't flip :)
```suggestion
	if (!flipCoin())
```